src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 package codeu.controller;
-
+import codeu.model.data.User; // unused for now
```

Comment:
Remove this line, it's a result of the merge conflict.

Reply:
Done.